src/mono/mono/mini/interp/interp.c

```diff
@@ -8595,6 +8627,31 @@ interp_sufficient_stack (gsize size)
 	return (context->stack_pointer + size) < (context->stack_start + INTERP_STACK_SIZE);
 }
 
+gboolean
+interp_jit_call_can_be_supported (MonoMethod *method, MonoMethodSignature *sig)
+{
+	if (sig->param_count > 10)
+		return FALSE;
+	if (sig->pinvoke)
+		return FALSE;
+	if (method->flags & METHOD_ATTRIBUTE_PINVOKE_IMPL)
+		return FALSE;
+	if (method->iflags & METHOD_IMPL_ATTRIBUTE_INTERNAL_CALL)
+		return FALSE;
+	if (!mono_llvm_only && method->is_inflated)
```

Comment:
mono_llvm_only will have the same value at AOT compile time and runtime, right? or is it possible for the user to accidentally run the application with a different value?

Reply:
Should be. @vargaz Am I right?

Reply:
No, at aot time, its set by doing --aot=llvmonly, and its only accessible to aot-compiler.c. So this might need a bool argument passed to this function, where the interp will pass mono_llvm_only, and and the aot-compiler.c passes acfg-><something>.

Reply:
Fixed.